### ISSUE TITLE
<body class="single-post"> on pages?

### ISSUE THREAD

**Issue by IvanPr** [MohammedKaludi/ampforwp-custom-theme#11]
The single.php has <body class="single-post"> hard-coded and sets 'single-post' even on pages...

**Comment by IvanPr** [comment]
<body class="single-post">

should be changed to:
<body class="single-post <?php
  if( is_page() )   echo 'amp-single-page';
  else                 echo 'amp-single';
?>">

**Comment by MohammedKaludi** [comment]
@IvanPr yes, that's a good suggestion, but I'm planning to create a function for body class, just like how currently WP handles the classes in the body tag and I'm sure this will be taken care in the future update.

Thank you.

**Comment by IvanPr** [comment]
I mean, it takes 30 seconds to edit the single.php file now for the rest of the valuable users :) and another several weeks to implement the functions in the future.

**Comment by ahmedkaludi** [comment]
@MohammedKaludi I think @IvanPr has a point here.

Can you just do that and commit? What do you say?

Regards,
Ahmed

**Comment by MohammedKaludi** [comment]
I have told @mayanksharma9033 to add the above code and he did the latest commit. 

But I will still be adding a function or some dynamic way to add classes in the body. So, I'm keeping this ticket open.

Mohammed Kaludi
